### PR TITLE
add min.insync.replicas config

### PR DESCRIPTION
**Pull request by switj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ $ terraform apply
 - `retention_ms` - the retention period in milliseconds for the topic
 - `cleanup_policy` - the clean up policy for the topic, for example compaction
 - `segment_bytes` - the segment file size for the log
-- `segement_ms` - the time after which Kafka will force the log to roll
+- `segment_ms` - the time after which Kafka will force the log to roll
+- `min_insync_replicas - the minimum number of insync replicas
```
Comment (badie) [inline README.md]: missing a backtick here. should be "\`min_insync_replicas\`"

**Comment by switj** [comment]
@badie I added a test, there aren't any existing for anything other than reading topics. I think if we're going to continue using this plugin we should probably add some.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ $ terraform apply
 - `retention_ms` - the retention period in milliseconds for the topic
 - `cleanup_policy` - the clean up policy for the topic, for example compaction
 - `segment_bytes` - the segment file size for the log
-- `segement_ms` - the time after which Kafka will force the log to roll
+- `segment_ms` - th:e time after which Kafka will force the log to roll
```
Comment (ebabani) [inline README.md]: typo `th:e`

**Comment by switj** [comment]
fixed the typo

**Pull request merged**